vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -147,42 +155,47 @@ Suppose you want to hide the page title on one page only. Here's how you can ach
 
 
 ## Overwrite CSS properties in selective components
-To overwrite CSS properties of selective components, pass an ID to the relevant component and target the right CSS property.
+To adjust CSS properties for specific components, like altering the appearance of a selected [Card][vizro.models.Card] rather than all Cards,
+you'll need to target the correct CSS selector based on the component hierarchy.
+If you're unfamiliar with CSS selectors, you can refer to this [tutorial](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selector_structure) for guidance.
 
-For more information, see this [CSS selectors tutorial](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selector_structure).
+Here's how you can do it:
 
-Let's say we want to change the background and font-color of one [`Card`][vizro.models.Card] instead of all existing Cards in the Dashboard.
-We can use the ID of the outermost Div to target the inner sub-components of the card. Note that all our components have an ID attached to the outermost Div,
-following the pattern `"{component_id}_outer"`.
+1. Assign a unique `id` to the relevant `Card`, for example: `Card(id="custom-card", ...)`
+2. Review the component's source code to find the appropriate CSS class or element you need to target.
 
-To achieve this, do the following:
+It's essential to understand the relationship between the targeted CSS class or element and the component assigned the `id`, for example:
 
-1. Pass a custom `id` to the relevant `Card`, for example: `Card(id="my_card", ...)`
-2. Take a look at the source code of the component to see which CSS Class you need to target such as `"card"`
-3. Use CSS selectors to target the right property by using the ID of the outermost Div `"my_card_outer"`
+* The `id` is provided to the `dcc.Markdown` component inside the `Card`.
+* The `dcc.Markdown` component is wrapped inside a parent container with the class name `"card"`.
+* The card text is wrapped inside a `<p>` element that is a child of the `dcc.Markdown` component.
 
+For instance, if you want to change the background color of a specific card, you need to target the parent
+HTML element with the CSS class `card`. If you want to change the font color, you need to target the `<p>` elements,
+which are children of the `dcc.Markdown` with the `id`.
```

Comment:
I think this is confusing because it's mixing the HTML tags with the underlying dash components. If someone looks at the Python source code then they will just see `dcc.Markdown` with no `<p>` anywhere. And if they look at the HTML source there's just HTML elements with nothing to do with `dcc.Markdown` there. 

I think somehow we need to make clear the following (doesn't have to all be done in this PR; probably will make more sense when we split this page up like we discussed the other day):
* some components generate a single HTML element with `id` corresponding to the model iD, so target directly with `#id` selector in CSS
* some components generate multiple HTML elements
  * "core" element will be a single HTML element with `id` corresponding to the model iD, so target direectly with `#id` selector in CSS
  * non-core element doesn't have an `id` explicitly, so target with some more complicated CSS selector involving `#id`
* in all cases you can work out what the right selector is by using Chrome dev tools or similar after you've set the appropriate model ID

Since CSS customisations are really frontend changes I think users should understand how to make the changes by looking at the HTML source rather than the Python source, in which case the fact that we have a `dcc.Markdown` is basically irrelevant and potentially confusing top mention, because it doesn't generate `<markdown>` HTML element.